src/main/java/gameframework/game/GameConfiguration.java

```diff
@@ -54,52 +54,90 @@ public GameConfiguration(int nbRows, int nbColumns, int spriteSize,
 					 : nbLives;
 	}
 
+	/**
+	 * Default constructor which construct a game configruation from default parameters
+	 */
 	public GameConfiguration() {
 		this(DEFAULT_NBROWS, DEFAULT_NBCOL, DEFAULT_SPRITESIZE, DEFAULT_NBLIVES);
 	}
 
 	// METHODS
-	
+
+	/**
+	 * @return the number of row from the game configuration
+	 */
 	public int getNbRows() {
 		return nbRows;
 	}
 
+	/**
+	 * @return the number of column from the game configuration
+	 */
 	public int getNbColumns() {
 		return nbColumns;
 	}
 
+	/**
+	 * @return the general size of sprite from the game configuration
+	 */
 	public int getSpriteSize() {
 		return spriteSize;
 	}
 
+	/**
+	 * @return the default number of lives feom the game configuration
+	 */
 	public int getDefaultNbLives() {
 		return nbLives;
 	}
 
+	/**
+	 * @return a new game canvas
```

Comment:
How should that canvas be used afterwards? Should one ask for a new one at each frame, or can we reuse that object?